scripts/lib/data-refiner.ts

```diff
@@ -144,20 +147,47 @@ function isPlausibleGeoGroup(geo_eligibility: string | undefined) {
   );
 }
 
-function parseDateToYear(input: string): string {
-  if (input.length === 4) return input;
-  let sep;
-  if (input.indexOf('/') !== -1) {
-    sep = '/';
-  } else if (input.indexOf('-') !== -1) {
-    sep = '-';
-  } else {
+/**
+ * This is for start and end dates of incentives. If the input is already in
+ * the required date format, it's returned as-is.
+ *
+ * Otherwise, we make an effort to parse a date from a few common formats.
+ * Ideally, though, spreadsheets should use our incentive date format directly
+ * (a superset of ISO 8601 dates).
```

Comment:
I think this is a good follow-up PR. If I'm understanding correctly, ajv is capable of validating using that regex pattern. If so, we can get rid of start_date_raw and end_date_raw and just make both date fields pass-through, and it's a CollectedIncentive schema validation error if the spreadsheet has an invalid date format.